lettre/style.css

```diff
@@ -0,0 +1,147 @@
+/*
+Theme Name: Lettre
+Theme URI: http://wordpress.com/theme/lettre/
+Author: Automattic
+Author URI: https://automattic.com/
+Description: A theme for writers that want to publish a newsletter using Jetpack.
+Requires at least: 6.1
+Tested up to: 6.1.1
+Requires PHP: 5.6
+Version: 1.1.0
+License: GNU General Public License v2 or later
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+Text Domain: lettre
+Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, featured-images, full-site-editing, block-patterns, rtl-language-support, sticky-post, threaded-comments, news
+
+Lettre WordPress Theme, (C) 2022 Automattic, Inc.
+Lettre is distributed under the terms of the GNU GPL.
+*/
+
+/*
+ * Font smoothing.
+ * This is a niche setting that will not be available via Global Styles.
+ * https://github.com/WordPress/gutenberg/issues/35934
+ */
+
+body {
+	-moz-osx-font-smoothing: grayscale;
+	-webkit-font-smoothing: antialiased;
+}
+
+/*
+ * Text and navigation link styles.
+ * Necessary until the following issue is resolved in Gutenberg:
+ * https://github.com/WordPress/gutenberg/issues/27075
+ */
+
+a {
+	text-decoration-thickness: 1px;
+	text-underline-offset: 0.25ch;
+	text-decoration-style: underline;
+}
+
+/*
+ * Button hover styles.
+ * Necessary until the following issue is resolved in Gutenberg:
+ * https://github.com/WordPress/gutenberg/issues/27075
```

Comment:
This issue is closed. Is there a more up to date issue that explains what is missing?

Reply:
Ah yeah it was closed earlier today, this CSS can now be removed!

Reply:
If we can't then we should open an issue for it :)

Reply:
I tried adding this via the `css` property, and it is possible, but I think it would be nice to have an `opacity` property in theme.json, so I opened an issue and updated the link in this file.